integration/types/datatype_test.rb

```diff
@@ -404,7 +402,7 @@ def test_can_insert_tuple_type_nil_values
 
       # Create the input
       params = [0]
-      DatatypeUtils.primitive_datatypes.each do |datatype|
+      DatatypeUtils.primitive_datatypes.each do
```

Comment:
I like this removal (and a few others like it I noticed). Make things cleaner. :)